config/routes.rb

```diff
@@ -66,6 +66,7 @@
         end
 
         resources :category_assessments, module: 'green_lanes', only: %i[index show create update destroy]
+        resources :themes, module: 'green_lanes', only: %i[index]
```

Comment:
Both these routes could be moved under a nested namespace (like news)

Reply:
done